server/src/main/java/org/apache/druid/server/coordination/ChangeRequestsSnapshot.java

```diff
@@ -24,7 +24,6 @@
 import com.google.common.base.Preconditions;
 
 import javax.annotation.Nullable;
-
```

Comment:
I ran optimize imports on this package and it removed some extra lines in this file and `SegmentChangeRequestDrop`.